prettyprinter-convert-ansi-wl-pprint/src/Prettyprinter/Convert/AnsiWlPprint.hs

```diff
@@ -110,17 +110,17 @@ toAnsiWlPprint = \doc -> case doc of
       where
         convertFg, convertBg, convertBold, convertUnderlining :: Old.Doc -> Old.Doc
         convertFg = case NewTerm.ansiForeground style of
-            Nothing -> id
-            Just (intensity, color) -> convertColor True intensity color
+            Just (NewTerm.Color16 intensity color) -> convertColor True intensity color
+            _ -> id
```

Comment:
Is there a reason for `_` vs. `Nothing` here? I don’t know `NewTerm`’s types, so making this explicit makes it clear which part you’re ignoring.